docs/students/faqs.md

```diff
@@ -36,28 +36,28 @@ No, most jobs ads don't ask for a degree. Some positions may ask for a degree, h
 
 ### Will I be able to use Khan Academy after the course finishes?
 
-Yes, all online course sites we will use through the course will be free including Khan Academy, Coursera, CodeAcademy. 
+Yes, all online course sites we will use through the course will be free, including Khan Academy, Coursera, FreeCodeCamp and CodeAcademy. 
 
 ### What should I do to avoid jamming all the information in my head?
 You should not be trying to memorize all the information you take in over the course. It is impossible to remember everything and no developer will ever work in this manner. As you move between roles and responsibilities, you may find yourself using some skills more than others, and over time your knowledge may become stronger and weaker in certain subjects about coding. This is okay! This is where the internet will help you. Yes, it is okay to research! One of the best resources out there is [Stackoverflow](https://stackoverflow.com/). Familiarize yourself with it early on.
 
 ### When I get stuck, what questions can I ask myself to figure out what the problem is?
 It will be hard at first to know what to ask yourself, but one way around this is to use pseudo code. Psuedo code refers to readable instructions and notes that help guide the developer through their code as they create each function or part of the code. When looking back over your code, this pseudo code can be helpful in finding out what your problem is. For example, let's say you build a calculator and one instruction you wrote for yourself is 'add two numbers together'. When you look back and realize you maybe used '-' instead of '+' between the two numbers, that instruction you left for yourself might give you a hint on what to look for. As you progress through the course, you will also learn how to 'debug' code in an effective way. Debugging simply means to fix a problem in your code.
 
-Another super effective way to get through difficulties is to explain your problem to a friend or colleague. It is a very typical issue in coding that youspend lots of time trying to solve something in vain, you become frustrated and finally cannot see the forest for the trees.  Explaining your problem to somebody else forces your mind to think the whole code through again line by line from top to bottom. This can help you understand the problem and spot where the mistake is hiding.
+Another effective way to get through difficulties is to explain your problem to a friend or colleague. It is a very typical in coding that you spend lots of time trying to solve something in vain; you become frustrated and cannot see the problem no matter how hard you look.  Explaining your problem to somebody else forces your mind to think the whole code through again line by line from top to bottom. This can help you understand the problem and spot where the mistake is hiding.
```

Comment:
Thanks for fixing it. A tiny bit fix, it'd sound better:
`It is very typical in coding when you spend ...`
rather than
`It is a very typical in coding that you spend ...`

Reply:
Ohh where my fancy idiom has gone :D , I know people might not get it.

Reply:
I dropped it since we had student feedback mentioned in the Retro about not understanding metaphors :p